fearless/Modules/NewWallet/SendFlow/ChooseRecipient/ChooseRecipientViewController.swift

```diff
@@ -0,0 +1,196 @@
+import UIKit
+import SoraUI
+
+final class ChooseRecipientViewController: UIViewController, ViewHolder {
+    typealias RootViewType = ChooseRecipientViewLayout
+
+    let presenter: ChooseRecipientPresenterProtocol
+
+    private var tableViewModel: ChooseRecipientTableViewModel?
+    private var locale = Locale.current
```

Comment:
Seems locale don't use in view controller

Reply:
Where are no optional String, it's required for search